frontend/server/src/Controllers/User.php

```diff
@@ -1329,6 +1329,47 @@ public static function getProfileImpl(
         return $response;
     }
 
+    public static function getProfileProgress(
+        \OmegaUp\DAO\VO\Users $user
+    ): float {
+        if (is_null($user->main_identity_id)) {
+            return 0;
+        }
+
+        $identity = \OmegaUp\DAO\Identities::getByPK($user->main_identity_id);
+        if (is_null($identity)) {
+            return 0;
+        }
+
+        $fields = [
+            'username' => !is_null($identity->username) ? 1 : 0,
+            'name' => !is_null($identity->name) ? 1 : 0,
+            'birth_date' => !is_null($user->birth_date) ? 1 : 0,
+            'gender' => !is_null($identity->gender) ? 1 : 0,
+            'scholar_degree' => !is_null($user->scholar_degree) ? 1 : 0,
+            'preferred_language' => !is_null($user->preferred_language) ? 1 : 0,
+            'verified' => $user->verified == '1' ? 1 : 0,
```

Comment:
```suggestion
            'verified' => $user->verified ? 1 : 0,
```

`$user->verified` es un [booleano de verdad](https://github.com/omegaup/omegaup/blob/0d1fd5bb321af06b2a554c7744c026d781fc43ce/frontend/server/src/DAO/VO/Users.php#L180-L185)!